carbonmark-api/src/routes/projects/get.utils.ts

```diff
@@ -54,88 +71,6 @@ export const getDefaultQueryArgs = async (
   };
 };
 
-/**
- * For a given project from polygon-bridged-carbon, returns array of pool price strings
- * Chooses between default and selective price.
- * Sorted cheapest first.
- */
-type ProjectBalances = Record<
-  string,
-  { balance: string; creditTokenAddress: string }
->;
-
-export const getDigitalCarbonTokenPrices = (
```

Comment:
So, this was the part where the prices where computed for the `/projects` endpoint. It was doing the same thing that was done in the `/projects/:id` endpointwith the `getProjectPoolPricesAndStats` but (i think) less accurately (hence the discrepencies in prices throughout the site). So I deleted it, refactored `getProjectPoolPricesAndStats` so it could accomodate both endpoints.